nano/node/node.cpp

```diff
@@ -3557,10 +3554,7 @@ void nano::active_transactions::request_confirm (std::unique_lock<std::mutex> &
 					// broadcast_confirm_req_base modifies reps, so we clone it once to avoid aliasing
 					if (!nano::is_test_network)
 					{
-						if (confirm_req_bundle.size () < max_broadcast_queue)
-						{
-							confirm_req_bundle.push_back (std::make_pair (i->election->status.winner, reps));
-						}
+						confirm_req_bundle.push_back (std::make_pair (i->election->status.winner, reps));
```

Comment:
`emplace_back` can be used, which can save a copy/move, i.e:
`confirm_req_bundle.emplace_back (i->election->status.winner, reps);`

Reply:
Thanks!

Now I guess we better make proper ordering for active transactions instead of random